tests/Client.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
+use Storyblok\Client;
+
 require '../vendor/autoload.php';
 
-$client = new \Storyblok\Client('Iw3XKcJb6MwkdZEwoQ9BCQtt');
+$client = new Client('Iw3XKcJb6MwkdZEwoQ9BCQtt');
```

Comment:
In the future will be nice to change thsi toke to a ENV_VAR here in the repo, to not expose the token to the end-users.

Reply:
next PR